network-api/networkapi/buyersguide/pagemodels/product_update.py

```diff
@@ -26,5 +30,19 @@ class Update(models.Model):
         blank=True,
     )
 
+    created_date = models.DateField(
+        auto_now=True,
+        null=True,
```

Comment:
we can actually remove this, and structure the migration as you have it right now, but with a third operation for `AlterField` that removes the `null=True`, as every product update will have a value assigned after `set_default_created_date` runs.

Reply:
```suggestion
```